apps/website/i18n/locales/en.json

```
@@ -104,12 +110,26 @@
     "character": "Character",
     "voiceActor": "Voice Actor",
     "notSpecified": "Not specified",
-    "modify": "Modify"
+    "modify": "Modify",
+    "voicedBy": "Voiced by",
+    "as": "As",
+    "noDubbingProjects": "No dubbing projects available",
```

Biome (2.5.6)

[error] The key noDubbingProjects was already declared.

(lint/suspicious/noDuplicateObjectKeys)